packages/react-native/Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -220,18 +220,22 @@ class TouchableHighlight extends React.Component<Props, State> {
         }
       },
       onPressIn: event => {
-        if (this._hideTimeout != null) {
-          clearTimeout(this._hideTimeout);
-          this._hideTimeout = null;
+        if (!Platform.isTV) {
```

Comment:
TODO: make sure this distinguishes correctly between touch events and TV remote control events.

Reply:
Done